src/main/resources/application.properties

```diff
@@ -17,3 +17,8 @@ logging.level.org.springframework.data.elasticsearch.core=DEBUG
 # Custom Elasticsearch index name
 #spring.data.elasticsearch.repositories.enabled=true
 spring.data.elasticsearch.properties.restClient.connectTimeout=5000
+
+# Custom URLs for Crawler
+url1=https://www.tutorialspoint.com/index.htm
+url2=https://www.w3schools.com
+url3=https://www.javatpoint.com
```

Comment:
Please try to read it in single variable rather than declaring variables for each endpoint